images/build/Dockerfiles/vso.focal.Dockerfile

```diff
@@ -112,9 +112,6 @@ RUN set -ex \
     && DOTNET_SDK_VER=$DOT_NET_60_SDK_VERSION \
        INSTALL_PACKAGES="true" \
        $imagesDir/build/installDotNetCore.sh \
-    && DOTNET_SDK_VER=$DOT_NET_50_SDK_VERSION \
```

Comment:
Any reason for us to remove .NET 5 from this image?

Reply:
That's a request from our team in order to support newer versions of languages/frameworks and drop support for older ones that are not actively maintained.

Hence, we want to remove DOTNET 5 from the image we use for the kitchensink.